etc/win-ci/crystal.iss

```diff
@@ -95,6 +95,7 @@ Root: HKA; Subkey: "Software\Classes\Applications\{#MyAppExeName}\DefaultIcon";
 Root: HKA; Subkey: "Software\Classes\Applications\{#MyAppExeName}\SupportedTypes"; ValueType: string; ValueName: ".cr"; ValueData: ""; Flags: uninsdeletekey
 
 [Icons]
+Name: "{group}\Crystal Book"; Filename: "https://crystal-lang.org/reference/master/index.html"

```

Comment:
Would it be easy to link to the appropriate version of the reference based on `MyAppVersionNum`?